toolset/setup/linux/webservers/resinJava8.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+RVER=4.0.41
+
+RETCODE=$(fw_exists ${IROOT}/resin-$RVER.installed)
+[ ! "$RETCODE" == 0 ] || { return 0; }
+
+fw_depends java8
+sudo cp -r $JAVA_HOME/include $JAVA_HOME/jre/bin/
+
+fw_get http://www.caucho.com/download/resin-$RVER.tar.gz
+fw_untar resin-$RVER.tar.gz
+cd resin-$RVER
+./configure --prefix=`pwd`
+make
+make install
+
+mv conf/resin.properties conf/resin.properties.orig
+cat $FWROOT/config/resin.properties > conf/resin.properties
+
+mv conf/resin.xml conf/resin.xml.orig
+cat $FWROOT/config/resin.xml > conf/resin.xml
+
+touch ${IROOT}/resin-$RVER.installed
```

Comment:
A little nit-pick: `resin-java8` would conform a bit better to install file conventions.

Convention is also to have the install name match the `.installed` name. There may be issues since `resinJava8` != `resin-4.0.41`. Not catastrophic, but this will cause the toolset to always install `resinJava8` even if it is already installed.

Recommendation: use `resin-java8` and touch `resin-java8.installed` on this commented line.